nimble/host/include/host/ble_gap.h

```diff
@@ -610,6 +610,11 @@ struct ble_gap_event {
              * like to use.  The application callback should fill this in.  By
              * default, this struct contains the requested parameters (i.e.,
              * it is a copy of 'peer_params').
+             *
+             * For BLE_GAP_EVENT_L2CAP_UPDATE_REQ, only connection interval
+             * min/max and connection length min/max can be changed.  If they
+             * are changed, they must remain within the original range.
```

Comment:
I would wrote, that if new interval is not in the range of the one provide by the peripheral, it will be ignored.